configure.ac

```diff
@@ -119,7 +119,6 @@ AC_DEFINE_UNQUOTED([GETTEXT_PACKAGE], ["$PACKAGE"], [Gettext package.])
 AM_COND_IF([MINGW],
   [pkgdatadir='${prefix}/data'],
   [pkgdatadir='${datarootdir}/geany'])
-AM_COND_IF([MINGW],[LIBS="$LIBS -liconv"])
```

Comment:
This came in https://github.com/geany/geany/pull/2938, seems unncessary now. Mingw build is fine without it.